DesignSafe-CI/portal
# DES-2082: Account Creation text update

## Overview: ##
Modification to a few notices for users related to account creation

## PR Status: ##


## Related Jira tickets: ##

* [DES-2082](https://jira.tacc.utexas.edu/browse/DES-2082)

## Summary of Changes: ##
Updated subtext on email for registration form
Updated text on registration_sucessful and email_reset pages
Updated text on some other pages

## Testing Steps: ##
Check urls
1. /account/registration-successful/
2. /account/password-reset/
3. /account/register/
4. /account/activate
Make sure you find the text as seen in the screenshots below.

## UI Photos:
Register Page:
<img width="691" alt="Screen Shot 2021-10-01 at 4 32 15 PM" src="https://user-images.githubusercontent.com/7683802/135688503-3d59ae9b-5e51-4f11-bcb4-625f933e742a.png">
<img width="694" alt="2-RegistrationPageTop" src="https://user-images.githubusercontent.com/7683802/140199336-685d097d-87d3-43a6-b2d2-6a6bd637fb4f.png">

Registration Successful Page:
<img width="1202" alt="4-RegistrationCompletePage" src="https://user-images.githubusercontent.com/7683802/140199391-38563230-16a3-429b-acb0-9e46f4a1b62f.png">

Password Reset Page:
<img width="1190" alt="Screen Shot 2021-10-01 at 3 28 51 PM" src="https://user-images.githubusercontent.com/7683802/135688543-38b0b623-3b71-48f1-8190-082abeabc4f2.png">

Account Activation Page:
<img width="1168" alt="5-AccountActivationPage" src="https://user-images.githubusercontent.com/7683802/140199488-f795b442-f27f-4328-9387-6aa9b830fbad.png">


## Notes: ##
views.py file modified because it errored out locally, and the change makes it viewable again.
